generators/python/src/fern_python/generators/fastapi/core_utilities/core_utilities.py

```diff
@@ -134,6 +135,17 @@ def copy_to_project(self, *, project: Project) -> None:
             ),
             exports={"serialize_datetime"},
         )
+        # Only copy enum.py when generating actual enum classes (not string literals)
+        if not self._use_str_enums:
```

Comment:
wrong, check out how this is handled in the python SDK. We need to consider both `use_str_enums` and `enum_type` config